WebRoot/js/zimbraMail/share/controller/ZmListController.js

```diff
@@ -976,7 +976,7 @@ ZmListController.prototype._initializeNavToolBar =
 function(view) {
 	var prevButton, nextButton;
 
-	if (view === 'TKL-main') {
+	if (view.includes('TKL')) {
```

Comment:
Notes: To handle left arrow key event when user is on Left/Right arrow navigation bar. We are providing View dropdown as prevButton such that focus will comes to this when focus is on left most enabled element of Left/Right navigation bar and user press left arrow key.